modules/renter.go

```diff
@@ -141,6 +141,8 @@ type HostDBEntry struct {
 	// ScanHistory is the set of scans performed on the host. It should always
 	// be ordered according to the scan's Timestamp, oldest to newest.
 	ScanHistory HostDBScans
```

Comment:
yeah, my bad. Typically the pattern is to name the sorting type `ByX`. So we'd write:
```go
sort.Sort(modules.ByTimestamp(entry.ScanHistory))
```
The name becomes pretty ugly though, since `ByTimestamp` is ambiguous -- it ought to be `HostDBScansByTimestamp` or something.